### PR TITLE
Don't file experimental crash types.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Review by Alan32Liu — APPROVED** [review]
LGTM

**Review by Navidem — APPROVED** [review]

File: src/clusterfuzz/_internal/crash_analysis/crash_analyzer.py [hunk 1]
```diff
@@ -414,3 +418,8 @@ def has_ubsan_error(stacktrace):
       return True
 
   return False
+
+
+def is_experimental_crash(crash_type):
+  """Return whether or not the crash type is experimental."""
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/crash_analysis/crash_analyzer.py:424]: nit: returns

File: src/clusterfuzz/_internal/tests/appengine/handlers/cron/triage_test.py [hunk 2]
```diff
@@ -388,3 +388,12 @@ def test_unrecovered_exception(self):
     testcase = data_handler.get_testcase_by_id(self.testcase.key.id())
     self.assertEqual('Failed to file issue due to exception: unrecovered',
                      testcase.get_metadata(triage.TRIAGE_MESSAGE_KEY))
+
+  def test_no_experimental_bugs(self):
+    """Test no exception."""
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/tests/appengine/handlers/cron/triage_test.py:393]: nit: "Tests"
And I'm not sure this description is correct.

Comment (oliverchang) [reply src/clusterfuzz/_internal/tests/appengine/handlers/cron/triage_test.py]: fixed!

**Pull request merged**